scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -1406,6 +1395,7 @@ static void init_task_ctx(struct task_struct *p, struct task_ctx *taskc)
 	u64 now = scx_bpf_now();
 
 	__builtin_memset(taskc, 0, sizeof(*taskc));
+	taskc->nr_cpus_allowed = bpf_cpumask_weight(p->cpus_ptr);
```

Comment:
I like this way of handling the nr_cpus_allowed, I'll probably even borrow it :smile: